_posts/2019-01-25-updates.markdown

```diff
@@ -0,0 +1,38 @@
+---
+layout: post
+title: Welcome to the New Way
+date: 2019-01-23  08:00:00
+categories: updates
+---
+__STATUS__:
+- current version: __3.60.0__
+- next scheduled release: __2/13/2019__
+
+Welcome to a new year, a new release, and a new set of development goals for the Ilios Project in the coming year! Among the key features identified as target milestones for our 2019-20 roadmap we are happy to note:
+
+*SEARCH*: We have begun a year-long initiative to begin the incorporation of deep search to the Ilios platform. We hope to be able to include search across the entire curriculum and its content  - from learning objectives to learning materials - and will keep you all apprised of progress in this area.
+
+*Session Linking*: Beginning with the next release, the session linking tools will be available for Ilios.
+
+*Configurable options*: We expect to be providing the ability to modify an array of different school level attributes, including the time-window for calendar integrations and dashboard displays, naming conventions, and more.
+
+*Selectives*: The ability to tag and group sessions as selectives within the curriculum.
+
+*Improved and extended rollover*: allowing the rollover of entire group trees, selective choice of attributes for rollover in a course, and better publishing control.
```

Comment:
```suggestion
*Improved and extended rollover*: This will allow the rollover of entire group trees. There will also be selective choice of attributes for rollover in a course. Finally, this will include better publishing control.
```